### PR TITLE
SPOI-10589 Update outdated information and fix typo

### PR DESCRIPTION
**Pull request by trusli**

### CODE REVIEW

File: docs/dtmanage.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ At any time, you can go back to the installation wizard from the Configuration T
 
 ## Develop Tab
 
-The development area of dtManage is mainly geared towards the creation, upload, configuration, and launch of DataTorrent applications. The development home can be viewed by clicking the “Develop” tab in the main navigation bar on the top of the screen. A prerequisite to using the development tools of the UI is an understanding of what Apex Application Packages are. For more information, see the [Application Packages Guide](https://www.datatorrent.com/docs/guides/ApplicationDeveloperGuide.html).
+The development area of dtManage is mainly geared towards the creation, upload, configuration, and launch of DataTorrent applications. The development home can be viewed by clicking the “Develop” tab in the main navigation bar on the top of the screen. A prerequisite to using the development tools of the UI is an understanding of what Apex Application Packages are. For more information, see the [Application Packages Guide](http://docs.datatorrent.com/application_development/).
```
Comment (amberarrow) [inline docs/dtmanage.md]: http://docs.datatorrent.com/application_packages/

File: docs/dtmanage.md [hunk 2]
```diff
@@ -119,20 +119,10 @@ To access the application package listing, click on the "Apps" link from the Dev
 - Delete the app package
```
Comment (amberarrow) [inline docs/dtmanage.md]: Line 116 above needs 'operators' changed to 'operations'

File: docs/operator_development.md [hunk 3]
```diff
@@ -285,9 +285,7 @@ We have the following configuration parameters:
 Code
 ----
 
-The source code for the tutorial can be found here:
-
-[https://github.com/DataTorrent/examples/tree/master/tutorials/operatorTutorial](https://www.google.com/url?q=https://github.com/DataTorrent/examples/tree/master/tutorials/operatorTutorial&sa=D&usg=AFQjCNHAAgSpNprHJVvy9GSjdlD1uwU7jw)
+The source code for the tutorial can be found [here](https://github.com/DataTorrent/examples/tree/master/tutorials).
```
Comment (amberarrow) [inline docs/operator_development.md]: Why is the last URL element "operatorTutorial" removed ?

File: docs/operators/kafkaInputOperator.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-KAFKA INPUT OPERATOR
+Kafka Input Operator
 =====================
 
-### Introduction: About Kafka Input Operator
+### Introduction
 
 This is an input operator that consumes data from Kafka messaging system for further processing in Apex. Kafka Input Operator is an fault-tolerant and scalable Malhar Operator.
```
Comment (amberarrow) [inline docs/operators/kafkaInputOperator.md]: an fault-tolerant => a fault-tolerant

**Comment by amberarrow** [comment]
Rest looks good. Please rebase after addressing above points so there are no conflicts.

File: docs/operator_development.md (hunk 3 above)
Comment (trusli) [inline docs/operator_development.md]: They are duplicates

**Comment by trusli** [comment]
Opened a separate pull request. https://github.com/DataTorrent/docs/pull/116

**Pull request closed without merging**